db/migrate/20250925162215_set_en_to_existing_forms.rb

```diff
@@ -0,0 +1,8 @@
+class SetEnToExistingForms < ActiveRecord::Migration[8.0]
+  class Form < ApplicationRecord
+  end
+
+  def change
+    Form.where(language: nil).update_all(language: %w[en])
+  end
+end
```

Comment:
I'm not sure we need this migration because the language column has always been not null with a default value of "en" so this migration has probably never been run.

The migration is also using the wrong value`%w[en]` is an array with a single "en" value. The language field expects text so it should be "en". Trying to set %[en] on language throws an error.

Reply:
I double checked and you're right the default value gets set anyway (for the right column which is `available_languages` and that is supposed to be an array) so I'm going to drop the commit that adds this migration